canonical-websites/www.ubuntu.com
# Add the new colour theory

## Done
Update the brand button to replicate to the positive button. Remove the link colour override.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Click about and see that brand/positive buttons are green and links are "Vanilla" blue

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2332
